web/src/beta/lib/core/engines/Cesium/useEngineRef.ts

```diff
@@ -167,6 +167,12 @@ export default function useEngineRef(
           cart.height,
         ];
       },
+
+      negatuvePiToPi: (angle: number) => {
+        const viewer = cesium.current?.cesiumElement;
+        if (!viewer || viewer.isDestroyed()) return;
+        return Math.abs(CesiumMath.negativePiToPi(angle)) > Math.PI / 86400;
+      },
```

Comment:
The return value seems to be boolean which does not match the naming of the function. If this function is supposed to be a simple export of CesiumMath.negativePiToPi then just return it and handle the other logic at where it been used. Otherwise please rename with other proper names.